### PR TITLE
manifest: Update TF-A to v2.10.4

### PR DESCRIPTION
**Pull request by ceolin**
Update Trusted Firmware A to the latest LTS release.

### CODE REVIEW

**Comment by ceolin** [comment]
@povergoing can you take a look please ? The current version is outdated and this one is the current LTS.

**Review by d3zd3z — DISMISSED** [review]
I'm ok with the change. Do we know of any code that uses this that could be used to test this? This change won't trigger most of CI to run.

File: west.yml [hunk 1]
```diff
@@ -332,7 +332,7 @@ manifest:
       groups:
         - tee
     - name: trusted-firmware-a
-      revision: 421dc050278287839f5c70019bd6aec617f2bbdb
+      revision: pull/4/head
```
Comment (dkalowsk) [inline west.yml:335]: should this be pulling head or a specific commit?

**Review by dkalowsk — COMMENTED** [review]
n/a

**Review by dkalowsk — COMMENTED** [review]
Overall I like this as a change for the LTS.  I thought we'd want a specific hash though

File: west.yml (hunk 1 above)
Comment (dkalowsk) [reply west.yml:335]: nm I  mis read the PR

**Comment by nashif** [comment]
@ceolin  can please point to the SHA now

**Comment by ceolin** [comment]
@dkalowsk Thanks ! Now it is using a hash.

@nashif done :)

**Review by dkalowsk — APPROVED** [review]

**Review by ithinuel — APPROVED** [review]

**Pull request merged**